src/main/kotlin/org/lsst/dax/albuquery/resources/Async.kt

```diff
@@ -161,24 +228,36 @@ class Async(val metaservDAO: MetaservDAO) {
             // FIXME: We're reasonably certain this will execute, execute a history task
             val queryTaskFuture = EXECUTOR.submit(queryTask)
 
+            // housekeeping
+            queryTask.phaseInfo.parameters = query
+            queryTask.phaseInfo.phase = "EXECUTING"
```

Comment:
I thought when requests were created they were supposed to stay in a pending state until the client says to put them in the running state?

5.1.3 Running a Query in http://www.ivoa.net/documents/TAP/20180416/PR-TAP-1.1-20180416.pdf

Reply:
Eventually that's how it should be, but SUIT specifically requested for this initial implementation that we execute the tasks immediately.